api/Console.json

```diff
@@ -0,0 +1,1048 @@
+            },
+            "firefox": {
+              "version_added": "8",
+              "notes": [
+                "Available in workers from 38 onwards"
+              ]
+            },
+            "firefox_android": {
+              "version_added": "8",
+              "notes": [
+                "Available in workers from 38 onwards"
+              ]
+            },
+            "ie": {
+              "version_added": "9"
+            },
+            "opera": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": false,
+            "deprecated": false
+          }
+        }
+      },
+      "dirxml": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/console/dirxml",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": true
+            },
+            "opera": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": false,
+            "deprecated": false
+          }
+        }
+      },
+      "error": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/console/error",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true,
+              "notes": [
+                "Alias (console.exception) not available"
+              ]
+            },
+            "chrome_android": {
+              "version_added": null,
+              "notes": [
+                "Alias (console.exception) not available"
+              ]
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "4",
+              "notes": [
+                "Substitution strings available from 9 onwards",
```

Comment:
Please add substitution strings as a separate sub feature here: 

```json
"substitution_strings": {
  "__compat": {
    "description": "Substitution strings",
    "support": {
```
In the tree this would get added here:
- api
  - Console
    - __compat
    - error
      - __compat
      - substitution_strings
    - info
      - __compat
      - substitution_strings
    - ...